src/index.js

```diff
@@ -79,7 +80,8 @@ export default class extends PureComponent {
     imgSrc: "",
     saveData: "",
     immediateLoading: false,
-    hideInterface: false
+    hideInterface: false,
+    refreshBackgroundImage: false
```

Comment:
Based on another comment, we would then loose the `refreshBackgroundImage` prop, and update every time the imgSrc changes.

```suggestion
    hideInterface: false
```